make/integrations/most.Makefile

```diff
@@ -4,6 +4,8 @@ else
   Q = @
 endif
 
+.NOTPARALLEL:
+
```

Comment:
@gillev-lb did you found a Jenkins build or other that really ran make in parallel?

Reply:
Yes, guy found and came to me with the issue.

Reply:
Nice. Weird that it wasn't noticed before.